craft_providers/actions/snap_installer.py

```diff
@@ -73,7 +73,7 @@ def validate_channel(cls, channel):
 
         :raises BaseConfigurationError: if channel is empty
         """
-        if channel == "":
+        if channel == "":  # noqa: PLC1901
```

Comment:
```suggestion
        if channel is not None and not channel:
```

I'm generally a fan of eliminating linter overrides, but I'm honestly not sure in this case which one I prefer.

Reply:
I'm 51/49 too - I think your's is slightly more explicit and clear.

Reply:
Especially since the `# noqa: PLC1901` will make me stop and have to think about what is "wrong" with the if statement